components/logged_in_navigation/components/you/index.jade

```diff
@@ -7,7 +7,7 @@
     )
 
 .Dropdown.js-dropdown
-  .DropdownSection
+  .DropdownSection.DropdownSection--expandable.js-expandable( data-key='you', data-state='active' )
     a.DropdownItem.DropdownItem--primary( href="/#{user.get('slug')}" )
```

Comment:
This link is getting the `preventDefault` treatment, no? Does it make sense to add a link to the profile directly? cc @danpia